### PR TITLE
TASK: Update used Symfony components to 2.8

### PR DESCRIPTION
**Pull request by cedricziel**
Since symfony follows semver (http://symfony.com/doc/current/contributing/code/bc.html) it's almost always possible to upgrade components within the given major without BC breaks.

This updates the used symfony components to 2.8, which also matches the most current [LTS](http://symfony.com/doc/current/contributing/community/releases.html#long-term-support-versions) version supported.

I would also bring a `^3.0` requirement to the table, but that one would mean a little more work and may be done in a second step.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -89,7 +89,16 @@
     "extra": {
         "installer-name": "Framework",
         "neos": {
-            "warning": "AUTOGENERATED FILE, ONLY MODIFY THE .composer.json IN THIS DIRECTORY AND RUN THE MERGE SCRIPT."
+            "warning": "AUTOGENERATED FILE, ONLY MODIFY THE .composer.json IN THIS DIRECTORY AND RUN THE MERGE SCRIPT.",
+            "package-key": [
```
Comment (cedricziel) [inline composer.json]: I'm not so sure about this passage. Please tell me if I am supposed to cut it out.

Comment (kitsunet) [inline composer.json]: It wouldn't matter much and I guess as such I prefer to leave it in as to avoid manual edititing of this file.

**Comment by kitsunet** [comment]
Tested locally, on Neos and tests. 👍

**Comment by cedricziel** [comment]
Thx for merging!

**Pull request merged**